.travis.yml

```diff
@@ -3,7 +3,8 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "6"
+  - "8"
+  - "10"
```

Comment:
Will this work with two versions specified? IIRC this telling CI which version of Node to use, so I'd think it should be one or the other, not both 🤔

Reply:
I see that Travis used this config to build the matrix with both 8 *and* 10. Nice 👌